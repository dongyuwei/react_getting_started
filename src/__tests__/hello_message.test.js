'use strict';

import React from 'react';
import renderer from 'react-test-renderer';
import HelloMessage from '../hello_message'

it('renders HelloMessage correctly', () => {
  const tree = renderer.create(<HelloMessage name="world" />).toJSON();
  expect(tree).toMatchSnapshot();
});
