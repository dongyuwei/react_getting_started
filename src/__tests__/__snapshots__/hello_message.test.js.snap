// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders HelloMessage correctly 1`] = `
<div>
  Hello 
  world
</div>
`;
